entry_types/scrolled/package/spec/editor/controllers/PreviewMessageController-spec.js

```diff
@@ -341,6 +341,27 @@ describe('PreviewMessageController', () => {
       postUpdateTransientContentElementStateMessage({id: 5, state: {some: 'state'}});
     })).resolves.toEqual({some: 'state'});
   });
+
+  it('sends CHANGE_EMULATION_MODE message to iframe on change:emulation_mode event on model', async () => {
+    const entry = factories.entry(ScrolledEntry, {}, {
+      entryTypeSeed: normalizeSeed({
+        sections: [{id: 1}]
+      })
+    });
+    const iframeWindow = createIframeWindow();
+    controller = new PreviewMessageController({entry, iframeWindow});
+
+    await postReadyMessageAndWaitForAcknowledgement(iframeWindow);
+
+    return expect(new Promise(resolve => {
+      iframeWindow.addEventListener('message', event => {
+        if (event.data.type === 'CHANGE_EMULATION_MODE') {
+          resolve(event.data);
+        }
+      });
+      entry.trigger('change:emulation_mode', entry.sections.first());
```

Comment:
**suggestion(if-minor)**: Trigger effect by changing attribute

In cases like this, it can be safer to perform the action that causes the event to be dispatched (i.e. changing the attribute) instead of triggering the event itself. That way we can be sure the actual data send with the event (if the handler depends on it) is used correctly and not just that from our fixture event.

Reply:
done